build.sh

```diff
@@ -22,4 +23,8 @@ inbm/install-tc.sh                          Installs inbm for Ubuntu or Debian
 inbm/uninstall-tc.sh                        Uninstalls inbm for Ubuntu or Debian
 inbm/Intel-Manageability.preview.tar.gz     Holds binary files for inbm
 inbm/LICENSE				    INBM license
+inbm-ucc/install-tc-ucc.sh                  Installs inbm in UCC mode for Ubuntu or Debian
```

Comment:
Are we creating new install scripts for UCC?

Reply:
@pillaiabhishek nope, this is just the normal TC install script with UCC_MODE forced on at the beginning. it's created automatically during final build.